### PR TITLE
Update quickstart section

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

File: doc/docs/modules/ROOT/pages/quickstart.adoc [hunk 1]
```diff
@@ -273,11 +276,11 @@ spark.read.format("org.neo4j.spark.DataSource")
 |"1"|"Jane Doe"
 |===
 
-As you can see, the struct returned by the query is made of strings
-that you can then be cast Spark's getters (ie: `getLong`).
+As you can see, the Struct returned by the query is made of strings
+that you can then be cast Spark's getters (i.e., `getLong`).
```
Comment (NataliaIvakina) [inline doc/docs/modules/ROOT/pages/quickstart.adoc]: I think, this sentence is confusing. It looks like wordy or something is missing

Comment (conker84) [inline doc/docs/modules/ROOT/pages/quickstart.adoc]:
```suggestion
As you can see, the Struct returned by the query is made of strings
that you can then cast by using Spark's getters (i.e., `getLong`).
```
@NataliaIvakina sounds better?

**Review by conker84 — COMMENTED** [review]
@NataliaIvakina thank you so much. Please look at my comment

File: doc/docs/modules/ROOT/pages/quickstart.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply doc/docs/modules/ROOT/pages/quickstart.adoc]: yes, sounds better, thank you

**Comment by conker84** [comment]
@NataliaIvakina thank you!

**Pull request merged**